### PR TITLE
Fixes a bug where the data-article-path was malformed and caused 404 links

### PR DESCRIPTION
**Pull request by fdoxyz**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Uses template literals to more clearly interpolate a complex string in buildArticleHTML.js

## QA Instructions, Screenshots, Recordings

1. Go to tag page (must have enough posts for infinite loading to kick in as this is what uses the `buildArticleHTML.js` method)
1. Scroll until infinite load populates more cards
1. Click on any card (title)
1. Go back
1. Click on any other card's white space (i.e. not the title link)

It should redirect correctly, unlike what we currently see for example in https://dev.to/t/javascript

## Added tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [x] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

random run

### CODE REVIEW

**Review by mstruve — APPROVED** [review]
Works great for me! I 100% agree we should write tests but I am OK with this going out as is. Either you can write them immediately after or if you want to make an Issue it would probably be something nice and little for someone to do as an EOY project or for a contributor.

File: app/assets/javascripts/utilities/buildArticleHTML.js [hunk 1]
```diff
@@ -267,70 +267,39 @@ function buildArticleHTML(article) {
...
-            ' +
-      searchSnippetHTML +
-      '\
-            <div class="crayons-story__bottom">\
-              <div class="crayons-story__details">' +
-      reactionsDisplay +
-      commentsDisplay +
-      '</div>\
+    return `<article class="crayons-story"
+      data-article-path="${article.path}"
```
Comment (fdoxyz) [inline app/assets/javascripts/utilities/buildArticleHTML.js:271]: Looks like `eslint` or any other of the pre-commit hooks reorganized this a bit for me. 

Not sure if it's as readable like this compared to wrapping the entire string in parenthesis and having the string start at the next line (271), that way it's all at the same indentation.

But that's what the tool suggests so it's okay I guess 🤔

**Review by Zhao-Andy — APPROVED** [review]
Nice! I find the interpolation easier to read 🤷‍♂️

**Comment by fdoxyz** [comment]
Sweet thanks @mstruve and @Zhao-Andy !

I think I may have a system spec I can use to cover this... If it gets too tricky I'll merge soon and follow up

**Comment by fdoxyz** [comment]
Okay nope, this spec hasn't been easy. I'm unable to load the cards from the infinite scroll. Merging the fix now and will follow up soon

**Pull request merged**